windows-installer/generate_sdl_agent_exe.ps1

```diff
@@ -199,3 +205,5 @@ cp $NSIS_UNZU_DLL $NSIS_UNICODE_PLUGIN_DIR
 ##############################
 
 & $NSIS_MAKE /DVERSION=$version $STACKDRIVER_NSI
+
+Set-PSDebug -Trace 0
```

Comment:
Just curious, is this a cleanup step that was missing before?

Reply:
This step wasn't there. If you're asking what happens without this step, then the powershell session that spawned the script inherits the trace flag, which makes it really annoying to use it interactively.

As an aside, I find that having the Trace output makes it really hard for me to see errors, which has caused me to miss problems while I was developing the script. I think it's a bad practice to leave these on in most submitted scripts.

Reply:
Would it be possible to add a comment that this is cleanup for interactive usage?

Reply:
I'm confused, why is the comment needed here and not where Trace is set to 1? How is the comment improving readability of the script?

Reply:
It wasn't very clear why changing some value that is intended to be script-scope as the last step in a script has any effect... But this is not a big deal either way; please treat as optional.

Reply:
It's not clear to me either, and if the script fails the command doesn't run. But when it runs, it's useful :) First thing I do when I work with this script now is to comment out the first Trace command, so I can actually see what the script is doing. YMMV.